man/mono.1

```diff
@@ -134,10 +134,58 @@ on demand by using the
 .I mono_install_load_aot_data_hook
 method.
 .TP
+.I direct-icalls
+.Sp
+When this option is specified, icalls (internal calls made from the
+standard library into the mono runtime code) are invoked directly
+instead of going through the operating system symbol lookup operation.
+This requires use of the
+.I static
+option.
+.TP
 .I direct-pinvoke
 .Sp
 When this option is specified, P/Invoke methods are invoked directly
 instead of going through the operating system symbol lookup operation.
+This requires use of the
+.I static
+option.
+.TP
+.I dwarfdebug
+Instructs the AOT compiler to emit DWARF debugging information. When
+used together with the nodebug option, only DWARF debugging
+information is emitted, but not the information that can be used at
+runtime.
+.TP
+.I full
+.Sp
+This compiles the code to run under the assumption all code is AOTed, IL is stripped and the JIT is unavailable at runtime. This feature is experimental and must run with a special "aot_full" BCL profile. This profile does not ship with the standard distribution of Mono, so you may need to build this profile yourself from the Mono source.
+.TP
+.I hybrid
+.Sp
+This compiles the code to run under the assumption all code is AOTed and IL is stripped, but the JIT *is* available at runtime. This feature is experimental and some parts of the standard library may not work properly.
```

Comment:
This description does not make much sense. I don't know what it does on runtime but it could be nothing.